### PR TITLE
Native short links in bosun

### PR DESCRIPTION
**Pull request by captncraig**

### CODE REVIEW

**Comment by captncraig** [comment]
Google link shortener not supported any longer. We could remove the config key if we wanted to make that a breaking change. 

Links now generate as {bosun-host}/s/123

File: cmd/bosun/conf/system.go [hunk 1]
```diff
@@ -539,10 +539,12 @@ func (sc *SystemConf) AnnotateEnabled() bool {
 // MakeLink creates a HTML Link based on Bosun's configured Hostname
 func (sc *SystemConf) MakeLink(path string, v *url.Values) string {
 	u := url.URL{
-		Scheme:   "http",
-		Host:     sc.Hostname,
-		Path:     path,
-		RawQuery: v.Encode(),
+		Scheme: "http",
```
Comment (kylebrandt) [inline cmd/bosun/conf/system.go]: hardcoded http probably not good. I think someone has a PR regarding this.

**Comment by kylebrandt** [comment]
@captncraig Can you expand on not supported anymore? In other words, what happens to people that using Google shortner and start using this branch?

**Comment by captncraig** [comment]
Not supported anymore as in, we don't bother making a google shortlink. Just make our own. The code to call google is removed entirely.

**Pull request merged**